### ISSUE TITLE
Disable bloom filters while restoring data

### ISSUE THREAD

**Issue by Michal-Leszczynski** [scylladb/scylla-manager#3788]
As discussed during the summit ([link to presentation](https://docs.google.com/presentation/d/171tNSRZUEYe9GC2RO1hFSWj4NGe3FsfkSRLp8tENvG4/edit#slide=id.g2c85a3b8d19_0_147)) and per https://github.com/scylladb/scylladb/issues/17972, SM should disable restored table `bloom filter` in the same way as `tombstone_gc`.
AFAIU, not doing so might result in unexpectedly large memory usage and slow down the process.

cc: @karol-kokoszka @tzach @raphaelsc

**Comment by Michal-Leszczynski** [comment]
Ref https://github.com/scylladb/scylla-enterprise/issues/4560#issuecomment-2296346862

**Comment by mykaul** [comment]
That's not true anymore, as we do ensure BF is resized if needed, during compaction. @denesb @avikivity?

**Comment by denesb** [comment]
> That's not true anymore, as we do ensure BF is resized if needed, during compaction. @denesb @avikivity?

Yes, and we also have a memory limit for bloom-filters, which defaults to 0.2. I don't think we need to cater for clusters which explicitly disable the defense mechanism.

**Comment by Michal-Leszczynski** [comment]
Not relevant anymore.
